tavla/app/(admin)/edit/[id]/components/Preview/index.tsx

```diff
@@ -8,17 +8,25 @@ import { TBoard, TOrganization } from 'types/settings'
 function Preview({
     board,
     organization,
+    landingPage,
 }: {
     board: TBoard
     organization?: TOrganization
+    landingPage?: boolean
```

Comment:
Skjønner tanken med å sende inn `landingPage` her, men siden det kun er stylingen som endres så kan det kanskje være lettere om vi bare kopierer over dette i `PreviewCarousel` i stedet? Også kan vi f. eks legge `classname="rounded p-4 bg-primary border border-secondary"` inn i `globals.css` for å bli brukt begge plasser. Tar gjerne en liten snakk på denne